src/components/ViewComponent/ViewComponent.test.ts

```diff
@@ -3,29 +3,63 @@ import { shallowMount } from '@vue/test-utils';
 import ViewComponent from '@/components/ViewComponent/ViewComponent.vue';
 
 describe('ViewComponent', () => {
-  it('Render title prop', () => {
-    const title = 'Test Title'
-    const wrapper = shallowMount((ViewComponent as any), {
-      props:
-      {
-        title: title,
-        loading: false,
-      }
-    })
-    expect(wrapper.find('.header-container').text()).toContain(title)
-  })
+    const title = 'Test Title';
 
-  it('Render default slot content', () => {
-    const wrapper = shallowMount((ViewComponent as any), {
-      slots: { default: '<div class="my-slot-content"></div>' }
-    })
-    expect(wrapper.find('.my-slot-content').exists()).toBe(true)
-  })
+    it('Render title prop', () => {
+        const wrapper = shallowMount(ViewComponent as any, {
+            props: {
+                title: title,
+                loading: false,
+            },
+        });
+        expect(wrapper.find('.header-container').text()).toContain(title);
+    });
 
-  it('Render action slot content', () => {
-    const wrapper = shallowMount((ViewComponent as any), {
-      slots: { actions: '<div class="my-actions-content"></div>' }
-    })
-    expect(wrapper.find('.my-actions-content').exists()).toBe(true)
-  })
-})
+    it('Render title prop and is loading', () => {
+        const wrapper = shallowMount(ViewComponent as any, {
+            props: {
+                title: title,
+                loading: true,
+            },
+        });
+        expect(wrapper.find('.header-container').text()).toContain(title);
```

Comment:
Here it is fine to check for the string literal since we set it explicitly for the test